src/main/java/com/github/technus/tectech/thing/metaTileEntity/multi/GT_MetaTileEntity_EM_EyeOfHarmony.java

```diff
@@ -1487,34 +1502,74 @@ private void outputFluidToAENetwork(FluidStack fluid, long amount) {
     public String[] getInfoData() {
         ArrayList<String> str = new ArrayList<>(Arrays.asList(super.getInfoData()));
         str.add(GOLD + "---------------- Control Block Statistics ----------------");
-        str.add("Spacetime Compression Field Grade: " + BLUE + spacetimeCompressionFieldMetadata);
-        str.add("Time Dilation Field Grade: " + BLUE + timeAccelerationFieldMetadata);
-        str.add("Stabilisation Field Grade: " + BLUE + stabilisationFieldMetadata);
+        str.add(
+                "Spacetime Compression Field Grade: " + EOH_TIER_FANCY_NAMES[spacetimeCompressionFieldMetadata]
+                        + RESET
+                        + " ("
+                        + YELLOW
+                        + spacetimeCompressionFieldMetadata
+                        + RESET
+                        + ")");
+        str.add(
+                "Time Dilation Field Grade: " + EOH_TIER_FANCY_NAMES[spacetimeCompressionFieldMetadata]
+                        + RESET
+                        + " ("
+                        + YELLOW
+                        + spacetimeCompressionFieldMetadata
+                        + RESET
+                        + ")");
+        str.add(
+                "Stabilisation Field Grade: " + EOH_TIER_FANCY_NAMES[spacetimeCompressionFieldMetadata]
+                        + RESET
+                        + " ("
+                        + YELLOW
+                        + spacetimeCompressionFieldMetadata
+                        + RESET
+                        + ")");
         str.add(GOLD + "----------------- Internal Fluids Stored ----------------");
         validFluidMap.forEach(
                 (key, value) -> str.add(BLUE + key.getLocalizedName() + RESET + " : " + RED + formatNumbers(value)));
         if (recipeRunning) {
             str.add(GOLD + "---------------------- Other Stats ---------------");
             str.add("Recipe Success Chance: " + RED + formatNumbers(100 * successChance) + RESET + "%");
-            str.add("Recipe Yield: " + RED + formatNumbers(100 * recipeYieldCalculator()) + RESET + "%");
+            str.add("Recipe Yield: " + RED + formatNumbers(100 * yield) + RESET + "%");
             str.add("Astral Array Fabricators detected: " + RED + formatNumbers(astralArrayAmount));
             str.add("Total Parallel: " + RED + formatNumbers(parallelAmount));
-            str.add("EU Output: " + RED + formatNumbers(outputEU_BigInt) + RESET + " EU");
+            str.add("EU Output: " + RED + toStandardForm(outputEU_BigInt) + RESET + " EU");
+            str.add("EU Input: " + RED + toStandardForm(usedEU) + RESET + "EU");
```

Comment:
It would be nice to negate this imo, as it currently shows a negative number as input.